modules/openapi-generator/src/main/resources/php-symfony/Controller.mustache

```diff
@@ -230,7 +230,7 @@ class Controller extends AbstractController
                     // add custom format to request
                     $format = $mimeType;
                     $request->setFormat($format, $format);
-                    $currentFormat = $request->getContentType();
+                    $currentFormat = $request->getContentTypeFormat();
```

Comment:
symfony/http-foundation', '6.2', 
'The `getContentType` method is deprecated, use `getContentTypeFormat` instead.'